SoftLayer/managers/monitor.py

```diff
@@ -0,0 +1,85 @@
+"""
+    SoftLayer.monitor
+    ~~~~~~~~~~~~~~~~~~
+    Monitoring Manager/helpers
+
+    :license: MIT, see LICENSE for more details.
+"""
+
+
+from SoftLayer import utils
+
+
+class MonitoringManager(utils.IdentifierMixin, object):
+    """Manages monitoring for Hardware and Virtual Servers
+
+    :param SoftLayer.API.CLient client: an API client instance
+    :param string server_tpe: should be either
+                  'Hardware_Server' or 'Virtual_Guest'
+    """
+
+    def __init__(self, client, server_type='Hardware_Server'):
+        self.client = client
+        self.account = self.client['Account']
+        self.server = self.client[server_type]
+
+    def get_status(self, server_id):
+        """get the monitoring status of a server
+
+        :param int server_id: the id of the server
+
+        Definition of some of the monitoring status codes
+         0 = Down
+         1 = Warning
+         2 = OK
+         >2 = something strange happened
+        """
+
+        mask = "lastResult, subnet[virtualGuests,hardware]"
+        agent = self.server.getNetworkMonitors(id=server_id, mask=mask)
+        return agent
+
+    def list_hardware_status(self, **kwargs):
```

Comment:
I can live with how list_hardware_status() and list_guest_status() work currently, but not get_status() and `__init__()`